contracts/test/MockEzEthPool.sol

```diff
@@ -196,7 +196,7 @@ contract MockEzEthPool is
     {
         uint256[][] memory operator_tokens_tvls;
         uint256[] memory operator_tvls;
-        uint256 tvl = getTotalPooledEther();
+        uint256 tvl = getTotalPooledEther() + totalSupply;
```

Comment:
I don't think this is right. It should be:

```suggestion
        uint256 tvl = getTotalPooledEther() + totalSupply.mulDivDown(
            getTotalPooledEther(),
            getTotalShares()
        );
```

To convert from vault shares to base.

Reply:
Really, we want the share price to be: `getTotalPooledEther() / getTotalShares()`. With that in mind, we should solve:

```
tvl / totalSupply = getTotalPooledEther() / getTotalShares() 

                      =>
                      
tvl = totalSupply.mulDivDown(getTotalPooledEthere(), getTotalShares())
```

Reply:
thank you yeah that looks right